lighthouserc.js

```diff
@@ -16,6 +16,7 @@
 module.exports = {
   ci: {
     collect: {
+      // headful: true,
```

Comment:
did you mean to leave this here for easier local dev or was it meant to be cleaned up after our session? 😬

Reply:
Bit of both 😅 Can remove it to be honest, and we can pick it up in docs instead.

Reply:
I think `autorun --headful` does the same thing anyways.